src/block/VanillaBlocks.php

```diff
@@ -163,6 +163,7 @@
  * @method static DoubleTallGrass DOUBLE_TALLGRASS()
  * @method static DragonEgg DRAGON_EGG()
  * @method static DriedKelp DRIED_KELP()
+ * @method static Kelp KELP()
```

Comment:
This should be below `JUNGLE_WOOD()`

Reply:
These comments are not sorted alphabetically, are they?

Reply:
They are as you can see: `DOUBLE_TALLGRASS()`, `DRAGON_EGG()`, `DRIED_KELP()`, etc.

Reply:
You can generate the comment using `build/generate-registry-annotations.php`.